ovirt_imageio/client/_options.py

```diff
@@ -161,6 +161,14 @@ def add_sub_command(self, name, help, func, transfer_options=True):
                 type=option.type,
                 help=option.help)
 
+        cmd.add_argument(
+            "--insecure",
+            dest="secure",
+            action="store_false",
+            default=True,
+            help=("Do not verify server certificates and host name (not "
+                  "recommened)."))
```

Comment:
Why not add this as Option like other options?

    Option(
        name="secure",
        args=["--insecure"],
        config=True,
        action="store_false",
        ...
    ),

This way we can control if the option can be set in the config file - you added
it to the config file, but I don't think the code will check it, since it does
not know about this option.

I guess we will have to add support for action= to the Options, but it should
be easy to add.

Reply:
I tried, added the support for action and everything, but `action` cannot coexist with `type` in the `add_argument` call:
```
            cmd.add_argument(
                *option.args,
                dest=option.name,
                action=option.action,
                type=option.type,
                help=option.help)
```
...
```
TypeError: __init__() got an unexpected keyword argument 'type'
``` 
I could make a dictionary, add the command arguments conditionally, and then pass the dictionary to `add_argument` as kwargs. But that felt like an over complication for the one single option (i.e., `secure`) that would use it.

Reply:
Specifying action only when not None and type is bool using **kwargs dict should work.